ui/src/virtualconsole/vcmatrix.cpp

```diff
@@ -83,6 +83,8 @@ VCMatrix::VCMatrix(QWidget *parent, Doc *doc)
     m_startColorButton->setFixedSize(48, 48);
     m_startColorButton->setIconSize(QSize(42, 42));
 
+    m_startColor = 0xeeeeee;
```

Comment:
This is not right

Reply:
@mcallegari
What is the correct value?

Reply:
https://github.com/mcallegari/qlcplus/blob/master/engine/src/rgbmatrix.cpp#L67

You can retrieve colors from RGB Matrix in setFunction method.
When no function is set, colors are basically "none"